models/login_source.go

```diff
@@ -403,6 +404,20 @@ func LoginViaLDAP(user *User, login, password string, source *LoginSource, autoR
 
 	var isAttributeSSHPublicKeySet = len(strings.TrimSpace(source.LDAP().AttributeSSHPublicKey)) > 0
 
+	// Update User admin flag if exist
```

Comment:
Isn't this will be already updated in https://github.com/go-gitea/gitea/pull/8849/files#diff-455da54e97ad39fc570175e880682f83R454 and is actually needed to update if no `autoRegister` is set?

Reply:
@lafriks No

checked: if user exists `CreateUser` throw an error!